nature/benchmarks/linear_mapper_benchmark.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Linear Mapper Benchmarks."""
+from timeit import timeit
+import retworkx
+from qiskit_nature.mappers.second_quantization import LinearMapper
+from qiskit_nature.problems.second_quantization.lattice.models import IsingModel
+from qiskit_nature.problems.second_quantization.lattice import Lattice
+
+# pylint: disable=redefined-outer-name, invalid-name, attribute-defined-outside-init
+
+
+class LinearMapperBenchmarks:
+    """Linear Mapper Benchmarks."""
+
+    version = 1
+    timeout = 120.0
+    params = [50, 80, 100]
+    param_names = ["Number of nodes"]
+
+    def setup_cache(self):
+        """setup cache"""
+
+        second_q_ops_list = []
+        edge_arr = list((n**11) % 100 for n in range(100))
+
+        for index, _ in enumerate(self.params):
+            graph = retworkx.PyGraph(multigraph=False)
+            graph.add_nodes_from(list(range(self.params[index])))
+
+            count = 0
+            for i in range(self.params[index]):
+                for j in range(i + 1, self.params[index]):
+                    count += 1
+                    graph.add_edge(i, j, edge_arr[i])
```

Comment:
Is `count` used anywhere? Should it be used as an index in `edge_arr`?

Reply:
This can be removed. 
The index for `edge_arr` can be provided by `i` itself, which is in range of the number of graph nodes.

Reply:
On a second thought, if we use `count` as the index to `edge_arr`, we will have to produce a list of edge weights of length ~5k. By using `i`, we are assigning same weights to edges that originate from the same node.

Reply:
Yeah, that's fine.